### DISCUSSION TITLE
Queue is not working if I send a request from the root domain

### DISCUSSION THREAD

**Discussion by Bouhaddi** [spatie/laravel-multitenancy#487] [Q&A]
Hi Everyone,

I have jobs table in my landlord database, it works only if I send a request from a sub-domain  (tenant1.localhost.test). but it doesn't works if I send a request from localhost.test

Im using database driver ( like I said the queue works fine if I send a request from a sub-domain) the issue is only when I send a request from root domain

Is the issue have anything to do with jobs aware ? because it works fine when I switch it to false

**Comment by masterix21** [comment]
Hi, try to dispatch your job using `$tenant->execute(fn () => dispatch(new YouJob));`
